analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -238,17 +239,6 @@ def add_arguments_to_parser(parser):
                                     ', '.join(analyzer_types.
                                               supported_analyzers) + ".")
 
-    analyzer_opts.add_argument('--add-compiler-defaults',
```

Comment:
@gyorb When do we remove a deprecated option? Do we need to wait some releases or what is the process of it?